src/Stepper/HorizontalStep.jsx

```diff
@@ -0,0 +1,221 @@
+import React from 'react';
+
+import {
+  TouchRipple,
+} from '../ripples';
+
+import Avatar from '../avatar';
+
+import {getMuiTheme} from '../styles';
+
+const HorizontalStep = React.createClass({
+  propTypes: {
+    /**
+     * The width of step header, unit is % which passed from Stepper.
+     */
+    headerWidth: React.PropTypes.string,
```

Comment:
As this property is controlled by Stepper, please add `@ignore`: https://github.com/callemall/material-ui/wiki/Documentation#internal-properties